2025/selection-process.md

```diff
@@ -0,0 +1,81 @@
+
+There are two steps to this phase:
+1. Proposal refinement: Oct 4th, 2024 to Oct 10th, 2024 - feedback would be provided to proposal authors on the Interop repo. 
+2. First round of feedback on proposals that do not meet the Interop project criteria would be provided to proposal authors on the Interop repo.
+
+
+### Phase 2: Ranking (Oct 18th to Oct 31st)
+
+Interop meeting dates: Oct 24th, Oct 31st
+
+[Visibility] All decisions, made in Interop team meetings during this phase, will be confidential. However, organizations can choose to publish their own priorities, for the Interop proposals, outside of the Interop program.
+
+In this stage, the focus would be on organizations picking areas where there is internal alignment on the specific feature or area being important enough to work on from a user/web developer perspective. Each organization will have different rubrics for prioritization. However, where applicable, organizations can choose to have any data used for prioritization be shared publicly.
+
+3. Each organization will select proposals for the prioritization discussion by putting a High/Low priority signal against each proposal where they have internal alignment to do the work. A low priority signal would indicate a willingness to support the proposal, if other organizations also deem it to be important. Organizations can also express any objections at this point.
+4. Any proposal that has at least 2 organizations expressing high priority and no objections, would be moved forward to the next phase of consensus building. Any organization that posts an objection would also outline the reasoning for it, to the Interop team. The ranking of proposals would be done asynchronously, ahead of the Interop meetings.
+
+
+### Phase 3: Prioritization (Nov 1st to Dec 5th)
+
+Interop meeting dates: Nov 7th, Nov 14th, Nov 21st, Dec 5th
+
+[Visibility] All decisions made in this phase will be confidential. 
+
+In this stage, each organization will further prioritize proposals for a specific year (Interop 2025). This is more involved and a complex process where each organization weighs engineering priorities and available resources.
+
+5. From the list of proposals that get through to Phase 3, each organization will have an opportunity to further prioritize internally and build consensus within the Interop Team based on available signals.
+6. Organizations can also object to any proposal at this point.
+7. Any proposal that has support from 2 organizations and does not have an objection, will be considered included in Interop 2025. At the end of this phase, the expectation is that the Interop team will have a first draft of proposals that are included in Interop 2025.
```

Comment:
This voting process looks like it's effectively another iteration of the Phase 2 process, but without using the "High/Low priority" terminology.

Are there any actual procedural differences? If not perhaps the same terminology should be reused to prevent confusion.

Reply:
Makes sense to clarify. In a procedural sense, the High/Low signals from round 2 will carry forward into round 3. Organizations will have the option to change that signal in round 3.